parser/src/java/org/apache/hadoop/hive/ql/parse/HiveLexerParent.g

```diff
@@ -385,6 +385,7 @@ KW_DATACONNECTORS: 'CONNECTORS';
 KW_TYPE: 'TYPE';
 KW_URL: 'URL';
 KW_REMOTE: 'REMOTE';
+KW_SPEC: 'SPEC';
```

Comment:
Would this mean that columns called `spec` could only be used from now on using backticks?

Reply:
No. I added `KW_SPEC` as a non-reserved keyword.

Reply:
Great, thanks!